docs/modules/buf-compare.md

```diff
@@ -0,0 +1,21 @@
+# buf-compare
+
+[`buf-compare`](https://github.com/sindresorhus/buf-compare) recommends that you switch to native `Buffer.compare()` which has been available since Node.js v0.12.0
+
+# Alternatives
+
+## NodeJS
+
+```js
+const {Buffer} = require('node:buffer');
```

Comment:
We should probably use esm syntax here to be consistent with the rest of the repo

Reply:
That's what I get for copy/pasting lol